src/main/java/models/Card.java

```diff
@@ -11,23 +11,41 @@
 
 public class Card implements Serializable {
     public final int value;
-    public final Suit suit;
+    public americanSuit amSuit;
+    public spanishSuit spSuit;
 
     @JsonCreator
-    public Card(@JsonProperty("value") int value, @JsonProperty("suit") Suit suit) {
+    public Card(@JsonProperty("value") int value, @JsonProperty("suit") americanSuit amSuit, @JsonProperty spanishSuit spSuit)
+    {
         this.value = value;
+<<<<<<< HEAD
```

Comment:
You should remove these before creating the pull request. I will remove everything between the HEAD and the "======" as well as the long string of chars and numbers.